pacemaker.spec.in

```diff
@@ -490,6 +490,8 @@ exit 0
 %doc %{_mandir}/man8/pacemakerd.*
 %doc %{_mandir}/man8/stonith_admin.*
 
+%doc %{_datadir}/pacemaker/alerts
```

Comment:
Thanks, this will also make `rpm -i --excludedocs pacemaker*.rpm` save some additional bits :)